VContainer/Assets/VContainer/Runtime/Unity/VContainerSettings.cs

```diff
@@ -53,6 +53,7 @@ public static void LoadInstanceFromPreloadAssets()
             var preloadAsset = UnityEditor.PlayerSettings.GetPreloadedAssets().FirstOrDefault(x => x is VContainerSettings);
             if (preloadAsset is VContainerSettings instance)
             {
+                instance.RootLifetimeScope.DisposeCore();
```

Comment:
Thanks for the fix.

I wonder if we need to consider the case where Instance.RootLifetimeScope is null?

Reply:
Thank you for reviewing.
You are right, we need a null checking of the RootLifetimeScope.